test/test_experimenter.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from pypuf.simulation.arbiter_based.ltfarray import LTFArray
+from pypuf.experiments.experiment.logistic_regression import ExperimentLogisticRegression
+from pypuf.experiments.experimenter import Experimenter
+
+
+class TestExperimenter(unittest.TestCase):
+    def test_lr_experiments(self):
```

Comment:
Please user smaller parameter numbers here to decrease test runtime (currently 20s on my machine)